### ISSUE TITLE
Updating target linking on cloned scenes

### ISSUE THREAD

**Issue by cojoMan** [grandseiken/foundryvtt-multilevel-tokens#80]
I don't know how other people run their games, but I have a nicely set of scenes that I might run for multiple people. So when I run it for groupA, I rename it and add at the end in parathesis "(groupA)".

Now, if I make an area to teleport in out/on scene1 to scene2, then duplicate these to run for my groupB, from what I understand, since it is the case of multiple identifiers with the same name, one will be randomly picked (?!), and groupB might end up on either the new duplicated page I've created for them, or the original, and multiple jumps might take them to each in turn...I do understand I can modify the tags on these maps, but it just makes the process much longer depending on the number of waypoints. 

How would this be best handled ? 
Is there an option to FULLY disable the module for specific scenes ? If I'd have them disabled on the initial "base scene", I could clone it, then enabled them for the clone, and even though they'd have the same identifier, only "active" scenes would be looked up.

**Comment by grandseiken** [comment]
You are correct in your understanding of how this would function, and that this isn't a well-supported use-case currently.

I would have expected that the easiest way to run the same scenes for different groups would be to use foundry's import/export feature to copy the scenes to different _worlds_ (rather than have duplicates in one world), and/or to just duplicate the whole world, which would work without issues. Is that not an option?

**Comment by cojoMan** [comment]
That is - if cumbersome, an option indeed. Thanks for the reply.

The way I use it, duplicating the scenes "in-world" is to account for the fact that these change - and specifically for complex ones - you might find an error on the groupA-scene, and switch back right away and fix it on the original. Not to say that you don't know what your players are up to so you branch out to a different scene of the world and can duplicate the original on the spot if you don't have it ready, as opposed to logoff, logon, export, etc. Seems easier to duplicate in world rather than duplicate the world, which seems to be the next best option.  

Seems like I could do this in the main world, waypoints and whatnot, then when duplicating, go through every one of them and add (groupA) as a safety measure.
